azkaban-common/src/main/java/azkaban/server/Constants.java

```diff
@@ -22,4 +22,16 @@
 	public static final String AZKABAN_PRIVATE_PROPERTIES_FILE = "azkaban.private.properties";
 	public static final String DEFAULT_CONF_PATH = "conf";
 	public static final String AZKABAN_EXECUTOR_PORT_FILENAME = "executor.port";
+
+  public static final String AZKABAN_SERVER_LOGGING_KAFKA_GLOBAL_DISABLE = "azkaban.logging.kafka.globalDisable";
+  public static final String AZKABAN_SERVER_LOGGING_KAFKA_BROKERLIST = "azkaban.logging.kafka.brokerList";
```

Comment:
What do you think of a name like azkaban.server.logging.kafka.brokerList? This way the name signals that this is a server config.